types/react-draft-wysiwyg/tslint.json

```diff
@@ -0,0 +1,80 @@
+{ 
```

Comment:
For a new package this file should just be `{ "extends": "dtslint/dt.json" }` with no disables.